rviz_simulator/launch/simulate.launch

```diff
@@ -0,0 +1,8 @@
+<launch>
+  <rosparam command="load" file="$(find rviz_simulator)/config/initialize_simulator.yaml" />
+  <rosparam command="load" file="$(find rviz_simulator)/config/realsense_camera_intrinsics.yaml" />
+  <node pkg="rviz" type="rviz" name="rviz"/>
+  <node pkg="rqt_console" type="rqt_console" name="rqt_console"/>
```

Comment:
I have renamed this file as "simulate_debug.launch" because I use rqt often during testing.

i'll make a separate file named "simulate.launch" to use when not debugging

Reply:
Instead of doing that, add a command line arg (e.g. debug_mode), when if debug_mode is true, then you launch these, otherwise, do not. 
See [here](http://wiki.ros.org/roslaunch/XML) for if conditions in launch files (Section 3) and [here](http://wiki.ros.org/roslaunch/XML/arg) for args.

Reply:
Will do.